### ISSUE TITLE
Manual build triggered should be part of existing event

### ISSUE THREAD

**Issue by thajith** [screwdriver-cd/screwdriver#1442]
**What happened**:
When an Manual SD job is triggered as part of the pipeline, SD triggeres a new event.

**What you expected to happen**:
Can you have a build triggered so that we can have a track of the automated build and manuals builds as part of the same event ?

**How to reproduce it**:
Trigger a manual build

**Comment by jithin1987** [comment]
This brings up the question of the need to differentiate between re-running a job in an event vs running a detached workflow in the event. Former has to be a new event for sake of tracking and record keeping, latter case is also treated same, but having it in the same event leads to less cluttering and is pretty much the same effect as the 2 separate events we have today.

**Comment by drewfish** [comment]
Yeah unfortunately it can be somewhat common for builds to be a little flaky (e.g. because they talk to external systems which might be flaky). Being able to re-run a job within the same event would be nice.
